meta-oe/recipes-core/sdbus-c++/sdbus-c++_0.7.2.bb

```diff
@@ -0,0 +1,46 @@
+SUMMARY = "sdbus-c++"
+DESCRIPTION = "High-level C++ D-Bus library designed to provide easy-to-use yet powerful API in modern C++"
+
+SECTION = "libs"
+
+LICENSE = "LGPLv2.1"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/sdbus-cpp-${PV}/COPYING;md5=1803fa9c2c3ce8cb06b4861d75310742"
+
+inherit cmake pkgconfig systemd ptest python3native
+
+DEPENDS += "expat ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'systemd', 'meson-native ninja-native git-native gperf-native gettext-native util-linux glibc libcap', d)}"
+RDEPENDS_${PN} = "${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'libsystemd', 'glibc libcap', d)}"
```

Comment:
I think hardcoding dependency on 'glibc' is not right. Did you mean something else here ?

Reply:
In this case sdbus-c++ build-depends and runtime-depends on librt library, which is AFAIK part of glibc. Do you have different suggestion in this regard?

Reply:
Librt is provided by c library and is not Glibc specific secondly dependency on Glibc are automatically added and are not required to be explicitly added

Reply:
So can I just remove glibc from `DEPENDS` and `RDEPENDS`?

Reply:
librt is provided by c library, but isn't glibc an implementation of c library for Linuxes?

Reply:
glibc is one of many, there is musl, there uclibc and maybe more. in OE we actively support musl and glibc, so hardcoding a libc dependency is not preferred unless its absolutely required then also we use COMPATIBLE_HOST variable to control the recipes for targets

Reply:
OK, glibc was removed.